src/views/Timeline.vue

```diff
@@ -90,7 +90,6 @@
             >
               <v-row
                 class="fill-height ma-0"
-                align="start"
```

Comment:
@vickywane can you please explain the result of changing from `align` to `justify`? Can you attach examples of before and after that show clearly the difference. Thank you.

Reply:
This was unintentional change